### PR TITLE
Donors: create dashboard DonorSchool model

### PR DESCRIPTION
**Pull request by breville**
With `donor-school.csv` in place from https://github.com/code-dot-org/code-dot-org/pull/30577, we can ingest it into a new DonorSchool model in the dashboard.

As with https://github.com/code-dot-org/code-dot-org/pull/30248, it's slightly unusual to have a Pegasus file be the source of data for a model in the Dashboard, but this keeps donor-related data in Pegasus gsheets.

This also fixes seeding of both `Donor` and `DonorSchool` by adding the seeding tasks to normal seed executions.

### CODE REVIEW

File: dashboard/app/models/donor_school.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+# A donor-school record sourced from the Pegasus CSV.
+class DonorSchool < ApplicationRecord
+  include Seeded
+
+  def self.setup
+    puts "in setup!"
```
Comment (islemaster) [inline dashboard/app/models/donor_school.rb]:
```suggestion
```

**Review by islemaster — APPROVED** [review]
Looks great!

**Review by islemaster — COMMENTED** [review]
Looks like drone failed to read the CSV at `pegasus/data/cdo-donor-schools.csv`.  Is there a missing `v3` in there somewhere?

**Review by maddiedierker — APPROVED** [review]
💰 🏫

**Review by clareconstantine — APPROVED** [review]

**Review by agealy — APPROVED** [review]

**Comment by breville** [comment]
@islemaster We just had to make sure there was a CSV available for seeding by merging https://github.com/code-dot-org/code-dot-org/pull/30577 and then scooping the new CSV on staging.

**Pull request merged**